html-aam/roles.tentative.html

```diff
@@ -18,6 +18,19 @@
 
 <input type="checkbox" switch data-testname="el-input-checkbox-switch" data-expectedrole="switch" class="ex">
 
+<!--
+  These thead, tbody, and tfoot role tests are pending spec discussion.
+  See https://github.com/w3c/html-aam/issues/474
+-->
+<table>
+  <thead data-testname="el-thead" data-expectedrole="rowgroup" class="ex">
+  </thead>
+  <tbody data-testname="el-tbody" data-expectedrole="rowgroup" class="ex">
+  </tbody>
+  <tfoot data-testname="el-tfoot" data-expectedrole="rowgroup" class="ex">
+  </tfoot>
+</table>
```

Comment:
I expect the lack of rows and cells invalidates the table context, including any rowgroup contexts inside it... WebKit, for example, won't expose malformed tables as tables in the accessibility tree, because that would force each assistive technology to do the error handling.

In order for the test to be valid, adding rows and cells to each row group. 

```suggestion
<table>
  <thead data-testname="el-thead" data-expectedrole="rowgroup" class="ex">
    <tr>
      <th>a</th>
      <th>b</th>
      <th>c</th>
    </tr>
  </thead>
  <tbody data-testname="el-tbody" data-expectedrole="rowgroup" class="ex">
    <tr>
      <th>1</th>
      <td>2</td>
      <td>3</td>
    </tr>
    <tr>
      <th>4</th>
      <td>5</td>
      <td>6</td>
    </tr>
  </tbody>
  <tfoot data-testname="el-tfoot" data-expectedrole="rowgroup" class="ex">
    <tr>
      <th>x</th>
      <th>y</th>
      <th>z</th>
    </tr>
  </tfoot>
</table>
```

Reply:
Oh, definitely - thank you. All good with me.